examples/discovery-search-app/cypress/plugins/cy-ts-preprocessor.js

```diff
@@ -0,0 +1,26 @@
+const wp = require('@cypress/webpack-preprocessor');
+
+const webpackOptions = {
+  resolve: {
+    extensions: ['.ts', '.js']
```

Comment:
did we change everything to `.ts`? if so, why do we need the `.js` extension?

Reply:
We don't 😄

Reply:
removing it now